File: Problem-Set-1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Positive Number? ");
    }
    while (n < 1 || n > 8);
    for (int row = 1; row <= n; row++)
    {
        for (int i = 1; i <= n - row; i++)
        {
            printf(" ");
        }
        for (int j = 1; j <= row; j++)
        {
            printf("#");
        }
        for (int l = 1; l <= 2; l++)
        {
            printf(" ");
        }
        for (int k = 1; k <= row; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Problem-Set-6/sentimental-mario-more/mario.py
while True:
    try:
        n = int(input("Positive Number? "))
        if n < 1 or n > 8:
            continue
        else:
            break
    except ValueError:
        continue

n = n + 1
for row in range(1, n):
    for space in range(1, n - row):
        print(" ", end="")
    for pyramid in range(1, row + 1):
        print("#", end="")
    for barrier in range(1, 3):
        print(" ", end="")
    for right_pyramid in range(1, row + 1):
        print("#", end="")
    print()
